### ISSUE TITLE
Prototype: unit properties

### ISSUE THREAD

**Issue by juliohq** [JulioYagami/Cosmic-Kimat#1]
Add the following properties [and possible parameters] to `unit.gd`:

**Fixed**:

- Nation [Keysian, Elkibari, Zugguru]
- Class [Cultural, Social, Personal]
- Race [Coal black, Emerald green, Golden yellow]

Core psychological traits:

- Behavior [Passive, Agressive, Passive-Agressive]
- Thought [Creative, Abstract, Practical]
- Emotion [Hysteric, Stoic, Empathetic]

**Variable**:

none yet.

**Comment by psxws** [comment]
that's not quite how race works but it's ok as a placeholder for right now. I'd suggest "Coal Black", "Emerald Green", "Golden Yellow" (note the double captions) as it looks nicer

**Comment by juliohq** [comment]
I did split the properties into **fixed** and **variable**.

**Comment by psxws** [comment]
Nono, the emotions are fixed as well. We have no variable properties being modeled as of yet. 

When a pawn is generated, they are assigned a behavior trait, a thought trait, & a emotion trait (one of each). Those represent the core aspects of that pawn's personality & do not change

**Comment by psxws** [comment]
also the cultural/social/personal division is a trait of the class but there's really 27 different classes (9 per nation). More on that on the design board on Trello (see in particular CLASSGEN for the rules)

**Comment by psxws** [comment]
We should keep track of what type of class it is (whether CUL, SOC, or PER) but that won't be shown to the player

**Comment by juliohq** [comment]
Are behavior, thought and emotion fixed properties? It won't change futurely?

**Comment by psxws** [comment]
Yes. These are the character's core psychological traits, not passing moods

**Comment by juliohq** [comment]
@psxws I'm gonna change it now. What are the variable properties else? Something like energy, health?

**Comment by psxws** [comment]
none so far, leave it open for the future. no need to add any right now but it'll be good to have the framework to do so later

**Comment by juliohq** [comment]
@psxws **All ok. As for it, nothing to do for now.**
